docusaurus-docs/jupyterlab-conda-store/introduction.md

```diff
@@ -1,7 +1,71 @@
 ---
-sidebar_position: 1
 title: Introduction
 description: Introduction to JupyterLab Extension.
 ---
 
 # conda-store JupyterLab extension
+
+A extension to use the [conda-store UI][conda-store-ui] - a React-based frontend for conda-store, within JupyterLab.
+
+## Install 📦
+
+1. Pre-requisites: `conda-store-server`, JupyterLab `>=` 3.0 and `<=` 4.0, and Python `>=` 3.8 installed.
+
+2. Install the extension:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
Is it standard to colocate imports with call sites in these kinds of docs?

Reply:
Well, I believe so? I've been using this style convention for a while now across projects. But'll try to verify.

Reply:
I was wrong! There are some colocated imports in the Docusaurus docs itself, but looks like most users define it at the top. I'll move this import to the topm and make a fresh PR to update this everywhere else in the docs.